ecllibrary/std/File.ecl

```diff
@@ -959,4 +963,30 @@ EXPORT PromoteSuperFileList(set of varstring superNames, varstring addHead='', b
 EXPORT varstring GetEspURL(const varstring username = '', const varstring userPW = '') :=
     lib_fileservices.FileServices.GetEspURL(username, userPW);
 
+
+ /**
+ * Returns the path to the default Drop Zone
+ *
+ *
+ * @return              A string containing the path to the default DropZone.
+ *                      If more than one Drop Zone
+ *                      process is defined then the first found will
+ *                      be returned; will return an empty string if a Drop Zone
+ *                      cannot be found
+ */
+EXPORT varstring GetDefaultDropZone() :=
+    lib_fileservices.FileServices.GetDefaultDropZone();
+
+ /**
+ * Returns the a dataset with full paths to all Drop Zones
```

Comment:
https://github.com/hpcc-systems/HPCC-Platform/pull/12933#discussion_r325674287

Looks like that comment wasn't addressed.